travisci/db_dump.sql

```diff
@@ -365,7 +365,8 @@ CREATE TABLE workflow (
     organization character varying(255) NOT NULL,
     path character varying(255),
     repository character varying(255) NOT NULL,
-    workflowname text
+    workflowname text,
```

Comment:
Interesting, forgot that ui2 was using a separate sql dump

Reply:
Me too